packages/devextreme/js/ui/tab_panel.js

```diff
@@ -252,6 +273,51 @@ const TabPanel = MultiView.inherit({
         this._focusTarget().attr('tabIndex', -1);
     },
 
+    _getTabsOrientation() {
+        const { tabsPosition } = this.option();
+
+        if([TABS_POSITION.top, TABS_POSITION.bottom].includes(tabsPosition)) {
```

Comment:
```
if(tabsPosition === TABS_POSITION.top || tabsPosition === TABS_POSITION.bottom) {
  // ...
} 
```
looks simplier for me

Reply:
My way is not difficult for understanding.